wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/Components/Reactions/ReactionToggle.swift

```diff
@@ -78,6 +86,12 @@ class ReactionToggle: UIControl {
             stackView.centerXAnchor.constraint(equalTo: centerXAnchor)
         ])
 
+        translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            widthAnchor.constraint(equalToConstant: 43),
```

Comment:
Should this size be the same for all phones and iPads?

Reply:
Not sure, I took this size from the existing collection view item size.  @agisilaos what do you think?

Reply:
I think the size is correct. We can leave it as it is for now and further discuss it during playtest today with Wolfgang!

Reply:
Actually, we need to make it dynamic and not fixed otherwise it won't grow when the labels grow. Currently it breaks when accessibility sizes increase.

Reply:
@johnxnguyen That's actually correct! Are you planning to fix that in this or separate PR @johnxnguyen?

Reply:
I fixed it in this PR